src/controllers/lobby.controller.js

```diff
@@ -7,6 +7,17 @@ async function getLobby(req, res) {
     res.status(500).json({ message: err.message });
   }
 }
+async function getMy(req, res) {
```

Comment:
I suggest renaming the getMy functions, for example in getPlayerInLobby or isPlayerInLobby

Reply:
Since getMy and all the other functions in this file are part of the lobbycontroller and a call for tem would be `lobbyController.getMy` i think it is not needed to have the "lobby" dublicated in the call

Reply:
I agree with @BenjaminHac-Edu  that it would be clearer if we renamed it to something like `getCurrentLobby` even if it contains duplicated info.